pom.xml

```diff
@@ -60,31 +52,32 @@
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
 
-        <spring-cloud.version>Greenwich.RC2</spring-cloud.version>
-
-        <spring.boot.version>2.1.1.RELEASE</spring.boot.version>
+        <spring-boot.version>2.1.1.RELEASE</spring-boot.version>
+        <spring-cloud.version>Finchley.RELEASE</spring-cloud.version>
+        <spring-cloud-starter-feign.version>1.4.6.RELEASE</spring-cloud-starter-feign.version>
+        <feign-httpclient.version>8.18.0</feign-httpclient.version>
 
-        <lombok.version>1.18.2</lombok.version>
-        <jackson.version>2.9.4</jackson.version>
-        <mockito.version>2.15.0</mockito.version>
-        <assertj.version>3.9.1</assertj.version>
+        <lombok.version>1.18.4</lombok.version>
         <commons-io.version>2.6</commons-io.version>
         <ant-style-path-matcher.version>1.0.0</ant-style-path-matcher.version>
-        <logback-classic.version>1.2.1</logback-classic.version>
+        <logback-classic.version>1.2.3</logback-classic.version>
         <logstash-logback-encoder.version>4.11</logstash-logback-encoder.version>
         <janino.version>3.0.6</janino.version>
-        <jgit.version>4.10.0.201712302008-r</jgit.version>
+        <jgit.version>5.2.0.201812061821-r</jgit.version>
         <commons-lang3.version>3.7</commons-lang3.version>
-        <jackson.version>2.9.8</jackson.version>
+        <jackson.version>2.9.7</jackson.version>
```

Comment:
why decreasing the version here ? I recently upgraded to latest (https://github.com/societe-generale/ci-droid-tasks-consumer/commit/7983a6f5e37728c434d92d6db8ebd2ddbd7dbafc) because there were some CVEs reported